docs/content/connectors/vitess-cdc.md

```diff
@@ -69,15 +67,15 @@ Connector Options
 ----------------
 
 
-<div class="&ldquo;highlight&rdquo;">
-    <table class="&ldquo;colwidths-auto">
+<div class="highlight;">
```

Comment:
```suggestion
<div class="highlight">
```
<img width="464" alt="image" src="https://github.com/ververica/flink-cdc-connectors/assets/34996528/61713727-db82-47ff-aae8-6653c8c068b0">

Is the code here necessary?